Rules/AvoidPositionalParameters.cs

```diff
@@ -54,7 +54,6 @@ public IEnumerable<DiagnosticRecord> AnalyzeScript(Ast ast, string fileName)
                 if ((Helper.Instance.IsCmdlet(cmdAst) || declaredFunctionNames.Contains(cmdAst.GetCommandName())) &&
                     (Helper.Instance.PositionalParameterUsed(cmdAst)))
                 {
-                    Console.WriteLine("Cmdlet or function call");

```

Comment:
Do you know that you can just attach the debugger to it and step through it by attaching to the PowerShell process? In Debug, the cmdlet even has a `-AttachAndDebug` to automatically attach to Visual Studio.

Reply:
I didn't know that. Thanks.
I removed debug line.